frontend/src/metabase/dashboard/selectors.ts

```diff
@@ -1,5 +1,6 @@
 import _ from "underscore";
 import { createSelector } from "@reduxjs/toolkit";
+import { createCachedSelector } from "re-reselect";
```

Comment:
We can't use `reselect` for this yet, right? Do we need to be on v5?

Reply:
I can't say exactly, I just saw that re-reselect is a part of our package.json and found its usage very straightforward for our use case. We indeed use reselect v4 that comes with RTK and v5 has more features and configurations, but I didn't dig into it